api/client/client.go

```diff
@@ -902,7 +902,7 @@ func (c *Client) GetKubeServices(ctx context.Context) ([]types.Server, error) {
 
 // getKubeServicesFallback previous implementation of `GetKubeServices` function
 // using `GetKubeServices` RPC call.
-// DELETE IN 10.0
+// DELETE IN 11.0.0
```

Comment:
Why do we have to bump these back another version?

Reply:
Bumped it back since there are other items that need to be deleted in 11 that depend on this. Tried to get rid of it and ran into issues with those so pushed it back to avoid confusion.

Reply:
Got it, thanks!